run.sh

```diff
@@ -1,34 +1,71 @@
 #!/usr/bin/env bash
 set -e -o pipefail
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+CLEAR='\033[0m'
 
-if [ -z "$1" ]
-  then
-    TENV_REGTEST="trezor-user-env-regtest"
-elif [[ $1 == --no-regtest ]]; then
-    TENV_REGTEST=""
-fi
-
+TENV_REGTEST="trezor-user-env-regtest"
 SYSTEM=$(uname)
+XHOST_ADDRESS=""
+SERVICE_NAME=""
+PULL=1
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
 
-if [[ $SYSTEM == Linux* ]]; then
+function usage() {
+  if [ -n "$1" ]; then
+    echo -e "$1";
+  fi
+  echo "Usage: $0 [-r --no-regtest] [-p --no-pull]"
+  echo "  -r, --no-regtest      Run tenv only, no regtest"
+  echo "  -p, --no-pull         Do not pull changes, neither git nor docker. Automatically set to True when not on 'master'"
+  echo ""
+  exit 1
+}
 
-    echo -e "Setup xhost for video device output"
-    xhost +
-    echo -e "Downloading latest images"
-    docker-compose -f ./docker/compose.yml pull trezor-user-env-unix $TENV_REGTEST
-    echo -e "Starting trezor-user-env"
-    docker-compose -f ./docker/compose.yml up trezor-user-env-unix $TENV_REGTEST
+# parse params
+while [[ "$#" > 0 ]]; do case $1 in
+  -r|--no-regtest) TENV_REGTEST=""; shift;;
+  -p|--no-pull) PULL=0; shift;;
+  -h|--help) usage; shift;;
+  *) usage "${RED}Unknown parameter passed: $1${CLEAR}\n"; shift;;
```

Comment:
It could be good to `exit 1` if someone inputs bad parameter - one can overlook this message and have bad expectations about the spawned application

Reply:
That already happens in the usage function doesn't it? 🤔 But that being said for --help we should probably return 0 right? 🤔

Reply:
That's correct, it returns 1 indirectly via `usage`. Help on its own probably should exit with 0, but probably with 1 in this case of bad argument :) But not a big deal

Reply:
3b9f9cc